custom_components/tuya_local/devices/ips_pro_heatpump.yaml

```diff
@@ -1,19 +1,54 @@
-name: IPS heatpump
+name: IPS Inverter Heatpump
```

Comment:
This should be moved under a "products" section linked to the product id of the device , and an unbranded name used at top level instead.